stacks/upload-api-stack.js

```diff
@@ -67,6 +67,8 @@ export function UploadApiStack({ stack, app }) {
        'GET /error':   'functions/get.error',
        'GET /version': 'functions/get.version',
        'GET /metrics': 'functions/metrics.handler',
+       // AWS API Gateway does not know trailing slash... and Grafana Agent puts trailing slash
+       'GET /metrics/{proxy+}': 'functions/metrics.handler',
```

Comment:
> A greedy path variable catches all child resources of a route. To create a greedy path variable, add + to the variable name—for example, {proxy+}. The greedy path variable must be at the end of the resource path.
– https://docs.aws.amazon.com/apigateway/latest/developerguide/http-api-develop-routes.html

Do we need a variable here? could we just do 
```suggestion
       'GET /metrics/': 'functions/metrics.handler',
```

Reply:
yes, it fails to deploy with error of invalid route name ending in `/`. also tried `/+` which also fails...